pkg/reconciler/kubernetes/tektontrigger/tektontrigger.go

```diff
@@ -134,14 +139,16 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tt *v1alpha1.TektonTrigg
 // transform mutates the passed manifest to one with common, component
 // and platform transformations applied
 func (r *Reconciler) transform(ctx context.Context, manifest *mf.Manifest, comp v1alpha1.TektonComponent) error {
-	instance := comp.(*v1alpha1.TektonTrigger)
+	trigger := comp.(*v1alpha1.TektonTrigger)
 	triggerImages := common.ToLowerCaseKeys(common.ImagesFromEnv(common.TriggersImagePrefix))
+	instance := comp.(*v1alpha1.TektonTrigger)
```

Comment:
we have 2 `TektonTrigger` obj here `trigger` and `instance` do we need to have both?

Reply:
No we don't need 2 obj
Updated :+1: